### PR TITLE
docs: Is update contribution guides

### PR DESCRIPTION
**Pull request by coldice**
This PR updates the contribution guidelines and technical documentation.

### Change scope

The changes in this Pull Request include (please tick all that apply):

* [ ] Small language/grammar fixes
* [ ] Small content fixes 
* [ ] Addition of new content
* [ ] Sample code/command updates
* [ ] Platform fixes
* [x] Other

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -48,24 +49,32 @@ The main VuePress configuration file is located at `.vuepress/config.js`.
...
+There is a hidden file (not linked in the main menu) published at [/feature-test](/feature-test/index.md), which demonstrates the use of all custom components used on this platform. This page is also available on the deployed website at [https://tutorials.cosmos.network/feature-test/](https://tutorials.cosmos.network/feature-test/). You can always refer to this page when you need a reference for a component integration.
 
 
 ## Images
 
 **Please note that images _must_ be linked using an absolute path!**
 
+### SVG Images
+
+A special note on svg images: They are currently not processed by the asset pipeline therefore svgs must be placed in a static asset folder (`.vuepress/public/`). Landingpage images are stored in the `lp-images` subfolder. To embed such an images, you can embed `/lp-images/go-lp.svg`, which then points to the file at `.vuepress/public/lp-images/go-lp.svg`.
```
Comment (xavierlepretre) [inline CONTRIBUTING.md:76]:
```suggestion
A special note on svg images: They are currently not processed by the asset pipeline therefore svgs must be placed in a static asset folder (`.vuepress/public/`). Landingpage images are stored in the `lp-images` subfolder. To embed such an image, you can embed `/lp-images/go-lp.svg`, which then points to the file at `.vuepress/public/lp-images/go-lp.svg`.
```

File: TECHNICAL-SETUP.md [hunk 2]
```diff
@@ -47,24 +52,24 @@ This sections lists technical details for content authors. For general informati
 
 ## Main config file
 
-The main config file is located at `.vuepress/config.js`.
+The main config file is located in `.vuepress/config.js`
```
Comment (xavierlepretre) [inline TECHNICAL-SETUP.md:55]:
```suggestion
The main config file is located in `.vuepress/config.js`.
```

File: TECHNICAL-SETUP.md [hunk 3]
```diff
@@ -83,12 +88,14 @@ An example configuration might look like this, where each module is defined as a
 }
 ```
 
+Where each module is defined as a children of the root.
```
Comment (xavierlepretre) [inline TECHNICAL-SETUP.md:91]:
```suggestion
Where each module is defined as a child of the root. 
```

File: TECHNICAL-SETUP.md [hunk 4]
```diff
@@ -121,19 +128,41 @@ patterns: [
...
 If you create a new folder named `myfiles`, add `"myfiles/*/*.md"` to the list.
 
-### Landing pages
+### Landingpages
 
-#### Main landing page
+There are different types of landingpages used on this platform:
 
-#### Sub landing page
+* The main landingpages, served under `/`, displays the content defined in [README.MD](README.MD), using the `ModuleLandingPage` layout.
```
Comment (xavierlepretre) [inline TECHNICAL-SETUP.md:135]:
```suggestion
* The main landingpage, served under `/`, displays the content defined in [README.MD](README.MD), using the `ModuleLandingPage` layout.
```

File: TECHNICAL-SETUP.md [hunk 5]
```diff
@@ -121,19 +128,41 @@ patterns: [
...
+* Module landingpages (Like the [what is cosmos lp](academy/1-what-is-cosmos/index.md)) are stored as `index.md` file in a subfolder.
```
Comment (xavierlepretre) [inline TECHNICAL-SETUP.md:136]:
```suggestion
* Module landingpages (Like [what is cosmos lp](academy/1-what-is-cosmos/index.md)) are stored as an `index.md` file in a subfolder.
```

File: TECHNICAL-SETUP.md [hunk 6]
```diff
@@ -121,19 +128,41 @@ patterns: [
...
 
-#### Module landing pages
+#### Main landingpage
 
+Please note that the main landingpage file (`README.MD`) contains all the content definitions ans settings in its markdown header (area enclosed between `---`) as well as the text for the main repo readme (content below the markdown header).
+
+#### Module landingpages
+
+Module landingpages can use implement the  `<card-module/>` for an optional display of children pages within that content folder.
```
Comment (xavierlepretre) [inline TECHNICAL-SETUP.md:144]:
```suggestion
Module landingpages can implement the `<card-module/>` for an optional display of children pages within that content folder.
```

**Review by xavierlepretre — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -3,16 +3,17 @@
...
 If the tutorial instructs you to download a specific version, that means that the code and the docs for each tutorial are tested, supported, and based on a specific version of the software. Be sure to download and use the right version.
 
 
 ## Contributing
 
-Thank you for helping us to create and maintain awesome tutorials.
+Thank you for helping us to create and maintain this awesome developer portal.
 
-- To set up your environment for success, follow the [technical set up](TECHNICAL-SETUP.md) guidelines.
+- To setup the portal lcoally, please follow the [Local setup and build](TECHNICAL-SETUP.md#local-setup-and-build) guide in the [TECHNICAL SETUP](TECHNICAL-SETUP.md) document. You might also want to read [further environment tips](TECHNICAL_SETUP.md#Environments) in that file.
```
[the remark this answers is not in the document]
Comment (coldice) [reply CONTRIBUTING.md:16]: @xavierlepretre @CitMC on the word _setup_ I have been following this: https://grammarist.com/spelling/set-up-vs-setup/ which points me to use set up here?

[the remark this answers is not in the document]
Comment (coldice) [reply CONTRIBUTING.md:16]: Agree on adding the second sentence part 👍

[the remark this answers is not in the document]
Comment (xavierlepretre) [reply CONTRIBUTING.md:16]: To set up is the infinitive form of the verb, so 2 words.

**Review by CitMC — APPROVED** [review]
Added minor edits

**Comment by CitMC** [comment]
@coldice I believe it is ready to be merged. Added some minor edits.

**Comment by CitMC** [comment]
@coldice isn't this ready to be merged?

**Pull request merged**